.readthedocs.yml

```diff
@@ -8,10 +8,14 @@ python:
     - method: pip
       path: .
 
+sphinx:
+  fail_on_warning: true
```

Comment:
Did https://github.com/pyrometheus/pyrometheus/commit/20edff4f89ebe9a74b0deabe24e01ea9260e8102 not work?

Reply:
`nitpicky` only creates more warnings, but these don't fail the build without `fail_on_warning` being enabled (similar to the -W/-n sphinx-build options).